.editorconfig

```diff
@@ -0,0 +1,11 @@
+root = true
+
+[*]
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[*.py]
+charset = utf-8
+indent_style = space
+indent_size = 4
```

Comment:
Are you sure that's the entire config? 
Also can we set line to 120-160 at least? 
And remove the new line on each bracket?

Reply:
The majority of the formatting will come from [black](https://github.com/psf/black). This is also now enforced with the new action. You can find plugins for black for whatever editor you use, check out their docs.

The original project stated the code style was loosely based on standards used within Google. I did some research and found many of Google's projects use Black because it has sensible defaults and removes any back and forth arguments about code style. I think this is a good fit for any open source project for that exact reason.

I am personally a big fan of Black's style, especially the 88char line length!

Reply:
From the personal experience, most teams are using pyink which is based on black and getting rid of those nested line wraps.

Reply:
Yeah, I saw pyink during my research. From their readme:

> We would love to adopt Black, but adopting it overnight is too disruptive to the thousands of developers working in our monorepo. We also have other Python tooling that assumes certain formatting, it would be a too big task to update them all at once. We decided to maintain a few local patches to Black as a medium-term solution, and release them as a separate tool called Pyink.
> 
> Pyink is intended to be an adoption helper, and we wish to remove as many patches as possible in the future.

I didn't consider it because there is no value in using a tool that itself wishes it didn't exist and will eventually converge with black anyway. Let's just skip the middle man.